### PR TITLE
LOON-419: Data corrections

### PR DESCRIPTION
**Pull request by rposkocil**

### CODE REVIEW

File: backend/src/main/kotlin/cz/loono/backend/Application.kt [hunk 1]
```diff
@@ -19,7 +20,7 @@ import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
 @EnableScheduling
 class Application
 
-//change to trigger github action
+// change to trigger github action
```
Comment (d35h) [inline backend/src/main/kotlin/cz/loono/backend/Application.kt:23]: We don't this comment, don't we?

File: backend/src/main/kotlin/cz/loono/backend/api/service/HealthcareProvidersService.kt [hunk 2]
```diff
@@ -258,12 +260,15 @@ class HealthcareProvidersService(
             houseNumber = houseNumber,
             city = city,
             postalCode = postalCode,
-            category = category.map(HealthcareCategory::value).filter { !removedCategories.contains(it) },
+            category = category.map(HealthcareCategory::value)
+                .ifEmpty { correctedCategory.map { HealthcareCategory::value } }
+                .filter { !removedCategories.contains(it) } as List<String>,
```
Comment (d35h) [inline backend/src/main/kotlin/cz/loono/backend/api/service/HealthcareProvidersService.kt:265]: Mb?
```suggestion
            category = category.map(HealthcareCategory::value)
                .ifEmpty { correctedCategory.map(HealthcareCategory::value) }
                .filterNot(removedCategories::contains)
```

File: backend/src/main/kotlin/cz/loono/backend/api/service/HealthcareProvidersService.kt [hunk 3]
```diff
@@ -274,17 +279,19 @@ class HealthcareProvidersService(
...
             postalCode = postalCode,
-            phoneNumber = phoneNumber,
+            phoneNumber = phoneNumber ?: correctedPhoneNumber,
             fax = fax,
             email = email,
-            website = website,
+            website = website ?: correctedWebsite,
             ico = ico,
-            category = category.map(HealthcareCategory::value).filter { !removedCategories.contains(it) },
+            category = category.map(HealthcareCategory::value)
```
Comment (d35h) [inline backend/src/main/kotlin/cz/loono/backend/api/service/HealthcareProvidersService.kt:287]: Same here?

File: backend/src/main/kotlin/cz/loono/backend/Application.kt (hunk 1 above)
Comment (rposkocil) [reply backend/src/main/kotlin/cz/loono/backend/Application.kt:23]: neni primergovany master, nebude tam :-)

**Review by d35h — APPROVED** [review]

**Pull request merged**